vitest.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    reporters: 'verbose',
+  },
+})
```

Comment:
It's better to add an empty line at the end of file.

```suggestion
})

```